frontend/src/components/EditableText.js

```diff
@@ -11,6 +11,11 @@ function EditableText({id, text, rubric, backend, setLastUpdate}) {
   const PASSAGE = new RegExp(`\\{${rubric}} ?([^{]+)`);
 
   let handleClick = () => {
+    let passages = document.getElementsByClassName('main col');
+    for (let passage of passages) {
+      passage.innerHTML = passage.innerHTML.replace('<highlighted>', '');
+      passage.innerHTML = passage.innerHTML.replace('</highlighted>', '');
```

Comment:
You could use a single replace by matching a regex instead of a string.

Reply:
Thanks for the advice, this is now fixed in the last commit.